<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FontSettingDialog</class>
 <widget class="QDialog" name="FontSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>469</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>字体设置窗口</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>字体</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>字体</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>字号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="checkBold">
        <property name="text">
         <string>粗体</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>字体效果预览</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="checkItalic">
        <property name="text">
         <string>斜体</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="checkUnderline">
        <property name="text">
         <string>下划线</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboFontSize">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>初号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小初</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>一号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小一</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>二号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小二</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>三号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小三</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>四号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小四</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>五号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小五</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>六号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>小六</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>七号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>八号</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>7.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>9</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>11</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>12</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>14</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>16</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>18</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>22</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>24</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>26</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>28</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>36</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>48</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>72</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QFontComboBox" name="comboFont">
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <property name="currentFont">
         <font>
          <family>黑体</family>
         </font>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>100</height>
         </size>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;文字测试样例&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Text Demo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>高级</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QSpinBox" name="doubleFontStretch">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4000</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QCheckBox" name="checkFixedWidth">
        <property name="text">
         <string>等宽字体</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="intFontWeight"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>字体粗细</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>字间距</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleFontSpace">
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>拉伸字体</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FontSettingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FontSettingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
